### PR TITLE
✨ added Error Component

### PR DESCRIPTION
**Pull request by rayanfer32**
## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor/Revamp ( rewrite or restructure code)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [x] Frontend
- [ ] Backend

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I ran `npm run format`/`yarn format` before commit
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [ ] I have updated `develop` and merged to my branch before submitting pull request
- [ ] My pull request generate no conflicts with `develop` or `main` branch
- [ ] I requested code review from other team members

## Related Issue

- none

## Proposed Changes

- none

## Additional Info

- none

## Screenshots

|              Original              |              Updated              |
| :--------------------------------: | :-------------------------------: |
|  |  |
|  |  |

### CODE REVIEW

File: src/components/Panel3/Panel3.jsx [hunk 1]
```diff
@@ -76,7 +77,7 @@ function Panel3({ blocks }) {
 
       setTableTxRowElements((prev) => [newRows, ...prev].slice(0, MAX_ROWS));
     } catch (err) {
-      console.error(err);
+      Log(err);
```
Comment (shrivatsabhat) [inline src/components/Panel3/Panel3.jsx:80]: It should be Error(err) rather Log(err) I think

**Review by shrivatsabhat — APPROVED** [review]
Error card is good, good to proceed

File: src/components/Panel3/Panel3.jsx (hunk 1 above)
Comment (rayanfer32) [reply src/components/Panel3/Panel3.jsx:80]: ok

Comment (rayanfer32) [reply src/components/Panel3/Panel3.jsx:80]: `Error` is a keyword 
image

**Pull request merged**